core/src/main/java/org/geysermc/geyser/translator/protocol/java/level/JavaLevelChunkWithLightTranslator.java

```diff
@@ -381,3 +494,39 @@ public void translate(GeyserSession session, ClientboundLevelChunkWithLightPacke
         }
     }
 }
+
+class ExtendedCollisionsStorage {
```

Comment:
This can be `static final class`, and you can move it into the LevelChunkWithLightTranslator code block.

Reply:
https://github.com/GeyserMC/Geyser/pull/3505/commits/4d12d0f7ad8991b9ac999fe1a73b34295441ed00#diff-be261e57e1fd314463d26d271b48e1b9d0d74b67e755b097f44c9037da7a4627L496-R564